blog/r-tidyfinance-0-4-0-new-data-downloads/index.qmd

```diff
@@ -0,0 +1,126 @@
+---
+title: "tidyfinance 0.4.0: new data downloads"
+author:
+  - name: Christoph Scheuch
+    url: https://www.linkedin.com/in/christophscheuch/
+    affiliations:
+      - name: Independent
+  - name: Stefan Voigt
+    url: https://www.voigtstefan.me/
+    affiliations:
+      - name: University of Copenhagen
+      - name: Danish Finance Institute
+  - name: Patrick Weiss
+    url: https://sites.google.com/view/patrick-weiss
+    affiliations:
+      - name: Reykjavik University
+      - name: WU Vienna University of Economics and Business
+date: "2024-08-30"
+description: tidyfinance 0.4.0 is now on CRAN. Discover the new data download options it includes. 
+image: thumbnail.png
+image-alt: Abstract illustration of a data flow pipeline with arrows and lines moving from financial icons like stock charts, currency symbols, and graphs towards a central tidy data frame. The background features a gradient of deep blues and greens, symbolizing finance and technology. The text ‘tidyfinance 0.4.0’ and a small CRAN logo are prominently displayed, with additional subtle financial icons integrated into the design. Created with DALL-E.
```

Comment:
minor thought: When introducing the package, you chose a DALL-E image which resembled a real world picture. All other blog posts are more stylized like this abstract illustration. Maybe it would be a nice idea to stick to that real-world style so the blog posts have a visual contrast between r-tidyfinance and general stuff.

Reply:
I don't see much pattern across DALL-E pictures tbh and it is really hard to enforce that. I wanted to create a picture of a warehouse with multiple pipelines that converge to a single pipeline, but DALL-E refused consistently.